backend/RPI_Smart_Calendar/Calendar/models.py

```diff
@@ -1,3 +1,45 @@
+import datetime
+from django.utils.translation import gettext_lazy as _
 from django.db import models
-
+from django.utils import timezone
+from django.views.generic import detail
+import django
 # Create your models here.
+class Week(models.Model):
```

Comment:
Why "Week" and "Day" are models? They are just numbers without any meanings. You may need to change the structure and the relations of the models.